replica/transition.go

```diff
@@ -6,16 +28,29 @@ import (
 	"github.com/renproject/hyperdrive/block"
 )
 
-// A TransitionBuffer is used to temporarily buffer `Transitions` that are not ready to be processed because of the
-// `State`. All `Transitions` are buffered against their respective `Height` and will be dequeued one by one.
+// A TransitionBuffer is used to temporarily buffer `Transitions` that
+// are not ready to be processed because of the `State`. All
+// `Transitions` are buffered against their respective `Height` and
+// will be dequeued one by one.
```

Comment:
What comment width are you formatting to? Happy to go with anything reasonable as long as we are all consistent.

Reply:
I picked 70 characters. In fact I try to keep everything under 70 characters because it lets me fit three splits on my monitor.